monitoring/ceph-mixin/dashboards_out/cephfs-overview.json

```diff
@@ -64,6 +64,7 @@
          "datasource": "$datasource",
          "description": "",
          "fill": 1,
+         "fillGradient": 0,
```

Comment:
Curious: was this due to some upgrade in grafonnet version?

Reply:
Yes exactly, since with this commit https://github.com/grafana/grafonnet-lib/commit/7ba2afd9f2f37a24178ee96a7d8944db2bd8db47 on grafonnet!

Reply:
Thanks!